### PR TITLE
fix: remove deprecated functionality

### PR DESCRIPTION
**Pull request by ThaUnknown**
**What is the purpose of this pull request? (put an "X" next to item)**

[ ] Documentation update
[ ] Bug fix
[ ] New feature
[X] Other, please explain:

**What changes did you make? (Give an overview)**
Removed functionality which was meant to be deprecated as of v1

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -81,21 +81,8 @@ class WebTorrent extends EventEmitter {
       this.peerId, this.nodeId, this.torrentPort
     )
 
-    if (this.tracker) {
-      if (typeof this.tracker !== 'object') this.tracker = {}
```
Comment (alxhotel) [inline index.js:85]: I think you missed this line

**Review by alxhotel — CHANGES_REQUESTED** [review]

File: index.js (hunk 1 above)
Comment (ThaUnknown) [reply index.js:85]: no? I just simplified it to 1 if statement

Comment (ThaUnknown) [reply index.js:85]: sure did

**Review by alxhotel — APPROVED** [review]
LGTM

**Review by DiegoRBaquero — APPROVED** [review]

**Pull request merged**